app/views/components/RadioOption.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package utils
+package views.components
```

Comment:
This is not a view component. This is why it was in `utils`.

Reply:
Agreed `components` might as well be called `utils` given everything that is in it. I just did the quickest thing to get it to compile

Reply:
I'd argue that "helpers that are only used by views" should go in the `views` package